Tests/testing/scs-compatible-test.yaml

```diff
@@ -1,15 +1,37 @@
 name: SCS Compatible
 url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Tests/scs-compatible.yaml
 iaas:
+  - version: v4
+    standards:
+      - name: Standard flavors
+        url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0103-v1-standard-flavors.md
+        check_tools:
+          - executable: ./iaas/standard-flavors/flavors-openstack.py
+            args: "./iaas/scs-0103-v1-flavors.yaml"
+      - name: Flavor naming
+        url: https://raw.githubusercontent.com/SovereignCloudStack/standards/main/Standards/scs-0100-v3-flavor-naming.md
+        check_tools:
+          - executable: ./iaas/flavor-naming/flavor-names-openstack.py
+            args: "--mand=./iaas/scs-0100-v3-flavors.yaml"
```

Comment:
So we call the script twice and it only tests one of the aspects each time?
0103: Is the list of mandatory SCS flavors complete?
0100: Are the names starting with `SCS-` all according to the syntax rules and do the discoverable properties match what the names claim?
The scripts has previously done both -- separating concerns here is good.
(Obviously in CI testing, cycle time is an important goal, so running the script only once doing both tests is an option if cycle time would be heavily impacted. This test is rather fast, so the cleaner separation of concerns wins over performance optimization ...)

Reply:
Right, see https://github.com/SovereignCloudStack/standards/pull/319#discussion_r1302938346

(except we are talking two separate scripts, not one script being called twice)